docs/reference/cluster/nodes-stats.asciidoc

```diff
@@ -971,6 +974,29 @@ Contains statistics about all shards assigned to the node.
 (integer)
 The total number of shards assigned to the node.
 =======
+
+`mappings`::
+(object)
+Contains statistics about the mappings for the node.
+This is not shown for the `shards` level, since mappings may be
+shared across the shards of an index on a node.
++
+.Properties of `mappings`
+[%collapsible%open]
+=======
+`total_count`::
+(integer)
+Number of mappings.
+
+`total_estimated_overhead`::
+(<<byte-units,byte value>>)
+Estimated overhead size of mappings.
```

Comment:
I think we can describe the computation here:

```suggestion
Estimated heap overhead of mappings on this node, which allows for 1kiB of heap for every mapped field.
```

(similarly for the `_in_bytes` variant)

Reply:
@DaveCTurner did this. Is it OK to mention the 1KiB per field since we might change that in the future? For this reason, I thought we could leave the description here "generic" enough.

Reply:
I think if we change how these figures are computed then we should also update these docs.

In contrast I think the sizing docs should be generic, saying "check the node stats for the estimated mappings overhead" rather than making the sizing guidance depend explicitly on the field count. That way the sizing docs will remain correct even as the overhead definition evolves.